CHANGELOG.md

```diff
@@ -2779,7 +2783,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Uncategorized
```

Comment:
nit: Past changelog entry could be a bit more descriptive

Reply:
This header is from years before I joined the team. Nobody has any context for any of these changes, so nobody is in a position to accurate judge whether they are changes, additions, removals, etc.